runtime/interpreter/account.go

```diff
@@ -111,9 +112,11 @@ func NewAuthAccountValue(
 	}
 
 	var str string
-	stringer := func(_ SeenReferences) string {
+	stringer := func(memoryGauge common.MemoryGauge, _ SeenReferences) string {
 		if str == "" {
-			str = fmt.Sprintf("AuthAccount(%s)", address)
+			common.UseMemory(memoryGauge, common.NewRawStringMemoryUsage(13))
```

Comment:
Maybe add this to the list of known memory usages up top. Same for `PublicAccount`, `AuthAccount.Contracts`, etc below

Reply:
Good idea! Added with 0b763c8a88be1f222c7e286b24f15a6a00db0f19